.ci/build-app-image.sh

```diff
@@ -4,28 +4,28 @@ set -e 	# Exit immediately upon failure
 : ${1?"Need to pass BASE_IMAGE as argument"}
 : ${2?"Need to pass TEST_APP as argument"}
 : ${3?"Need to pass TAG as argument"}
-: ${4?"Need to pass BRANCH as argument"}
+: ${4?"Need to pass VERSION as argument"}
 
 BASE_IMAGE=$1
 TEST_APP=$2
 TAG=$3
-BRANCH=$4
+VERSION=$4
 
 echo "[CI] Injecting Dockerfile to project $TEST_APP..."
-cd  $SAMPLES_REPO/samples/$TEST_APP
-echo "[CI] Checkout branch $BRANCH"
-git checkout $BRANCH
+if [[ -d $SAMPLES_REPO/samples/$VERSION/$TEST_APP ]]; then
+	cd  $SAMPLES_REPO/samples/$VERSION/$TEST_APP
+else
```

Comment:
minor suggestion: maybe another `-d ...` and `else` would fail with an error message?